cplusplus/VImage.cpp

```diff
@@ -93,7 +93,7 @@ negate( std::vector<double> vector )
 {
 	std::vector<double> new_vector( vector.size() );
 
-	for( unsigned int i = 0; i < vector.size(); i++ )
+	for( size_t i = 0; i < vector.size(); i++ )
```

Comment:
I suppose strictly this should be something like:

```suggestion
for( std::vector<double>::size_type i = 0; i < vector.size(); i++ )
```

What do you think?

Reply:
Done.